:dev terminal test commands don't work anymore (testSupport.include is null)

### Describe the bug

When invoking the terminal (`:`) while in `:dev` mode, the options to set the `test pattern` for includes and excludes don't seem to work (anymore).

Maybe I'm missing something but I don't seem to be able to include and exclude patterns once the dev mode has been started.

Example:

```
:
$ test pattern exclude .*UITest
java.lang.NullPointerException: Cannot invoke "java.util.regex.Pattern.pattern()" because "testSupport.include" is null
```

If I look into the source code of `io.quarkus.deployment.dev.testing.TestSupport`, then indeed `include` and `exclude` are `null`.

### Expected behavior

_No response_

### Actual behavior

_No response_

### How to Reproduce?

1. Have a project with different test naming strategies (e.g. `...Test`, `...UITest`, `...APITest`).
2. Run `mvn quarkus:dev`
3. Start the terminal (`:`)
4. Try to set the exclude and include patterns (maybe multiple times)


### Output of `uname -a` or `ver`

Linux squarelinux 6.1.7-arch1-1 #1 SMP PREEMPT_DYNAMIC Wed, 18 Jan 2023 19:54:38 +0000 x86_64 GNU/Linux

### Output of `java -version`

OpenJDK Runtime Environment Temurin-18.0.1+10 (build 18.0.1+10)

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

3.1.1.Final

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

Apache Maven 3.8.6

### Additional information

_No response_

🤔

Stacktrace:

```
2023-06-12 10:01:22,034 WARNING [org.aes.rea.Process] (Thread-34) Uncaught exception when executing the command: io.quarkus.deployment.dev.testing.TestTracingProcessor$ExcludePatternCommand@747571f0: java.lang.RuntimeException: java.lang.NullPointerException: Cannot invoke "java.util.regex.Pattern.pattern()" because "testSupport.include" is null
	at org.aesh.command.impl.Executions$ExecutionImpl.execute(Executions.java:246)
	at org.aesh.readline.Process.run(Process.java:80)
Caused by: java.lang.NullPointerException: Cannot invoke "java.util.regex.Pattern.pattern()" because "testSupport.include" is null
	at io.quarkus.deployment.dev.testing.TestTracingProcessor$ExcludePatternCommand.configure(TestTracingProcessor.java:323)
	at io.quarkus.deployment.dev.testing.TestTracingProcessor$TestSelectionCommand.doExecute(TestTracingProcessor.java:343)
	at io.quarkus.deployment.console.QuarkusCommand.execute(QuarkusCommand.java:25)
	at org.aesh.command.impl.operator.EndOperator.execute(EndOperator.java:41)
	at org.aesh.command.impl.Executions$ExecutionImpl.execute(Executions.java:222)
	... 1 more
```

--- a/core/deployment/src/main/java/io/quarkus/deployment/dev/testing/TestTracingProcessor.java
+++ b/core/deployment/src/main/java/io/quarkus/deployment/dev/testing/TestTracingProcessor.java
@@ -298,7 +298,7 @@ public class TestTracingProcessor {
 
         @Override
         protected void configure(TestSupport testSupport) {
-            testSupport.setPatterns(pattern, testSupport.exclude.pattern());
+            testSupport.setPatterns(pattern, testSupport.exclude != null ? testSupport.exclude.pattern() : null);
         }
     }
 
@@ -320,7 +320,7 @@ public class TestTracingProcessor {
 
         @Override
         protected void configure(TestSupport testSupport) {
-            testSupport.setPatterns(testSupport.include.pattern(), pattern);
+            testSupport.setPatterns(testSupport.include != null ? testSupport.include.pattern() : null, pattern);
         }
     }
 

